src/main/resources/assets/tconstruct/lang/zh_cn.json

```
@@ -2626,6 +2630,7 @@
   "recipe.tconstruct.modifier.not_enough_slot": "至少需要1个%s槽来添加该强化物",
   "recipe.tconstruct.modifier.not_enough_slots": "至少需要%s个%s槽来添加该强化物",
   "recipe.tconstruct.modifier.validate_slots": "这项改动留给工具的%s槽过少",
+  "recipe.tconstruct.modifier.min_level": "This recipe requires the tool to have at least %s level %s to apply",
```

```suggestion
  "recipe.tconstruct.modifier.min_level": "此配方需要至少%s级%s才可使用",
```
Haven't yet tested this part of translations, may need to change the order of formatters.